services/storage/src/simcore_service_storage/handlers.py

```diff
@@ -381,7 +385,14 @@ async def delete_file(request: web.Request):
         return {"error": None, "data": None}
 
 
-# Exclusive for simcore-s3 storage -----------------------
+@routes.post(f"/{api_vtag}/simcore-s3:access")
+async def get_or_create_temporary_s3_access(request: web.Request):
+    user_id = UserID(request.query["user_id"])
+    with handle_storage_errors():
+        s3_settings: S3Settings = await sts.get_or_create_temporary_token_for_user(
+            request.app, user_id
+        )
+        return {"data": s3_settings.dict()}
```

Comment:
I see the default values. Regarding my PR https://github.com/ITISFoundation/osparc-simcore/pull/2993 I would still not add this. Will enabled it once we decide on how to use STS